game.c

```diff
@@ -200,7 +198,7 @@ int get_input(level *lvl) {
         case 'r':
             rotate_inventory(lvl->player);
             inventory = inventory_string(lvl->player, MESSAGE_LENGTH);
-            snprintf(message, MESSAGE_LENGTH, "Your inventory contains: %s", inventory);
+            snprintf(message, MESSAGE_LENGTH, "Currently selected item -> %s", inventory);
```

Comment:
Felt this was clearer. Wanted to make like a [ ] box around the selected item but it looks like it's a character string at this point not an array of words.